### DISCUSSION TITLE
Shelly 1PM adjusting the value Total

### DISCUSSION THREAD

**Discussion by EDSTOBI** [arendst/Tasmota#15478] [Q&A] [answered]
Hello
I had to replace a Shelly 1PM and would like to correct the total value or set it to the last known value.
Whether via mqtt or the console.
Can someone please tell me the sintax to do this? With EnergyTotal <value> I did not get a change at the console

**Comment by sfromis** [comment, marked as answer]
EnergyTotal is what you should use, but beware that it adjusts midnight level, meaning that EnergyToday is added to the value displayed. You could zero EnergyToday, or you could subtract the value from the value you want to have in EnergyTotal after adjustment.
